### DISCUSSION TITLE
开启版面分析比不开启时，表格识别效果大大降低

### DISCUSSION THREAD

**Discussion by xgySTATISICT** [PaddlePaddle/PaddleOCR#12493] [Q&A]
使用PPStructure接口、自己fineturn的表格模型进行表格识别，不开启版面分析时，表格识别效果非常好，开启版面分析后，识别效果变得很差，表结构和文字识别都变差，请问这是什么原因
`table_engine = PPStructure(table_model_dir="table",
                           det_model_dir="det",
                           rec_model_dir="res",
                           # layout=False,
                           # table=False,
                           # ocr=True,
                           layout=True,
                           output=save_folder, show_log=True, recovery=False, return_ocr_result_in_table=True, use_gpu=False)`

**Comment by gl94** [comment]
猜测是因为你版面分析模型对表格位置的检测不准确,可以单独可视化版面分析模型看看效果

**Comment by AndrewDing616** [comment]
你好，可以咨询一下如何进行finetune吗，求！
